dags/openshift_nightlies/tasks/benchmarks/e2e.py

```diff
@@ -76,13 +94,13 @@ def _add_indexers(self, benchmarks):
                     self._add_indexers(benchmark)
 
     def _add_indexer(self, benchmark): 
-        indexer = StatusIndexer(self.dag, self.config, self.release, benchmark.task_id).get_index_task() 
+        indexer = StatusIndexer(self.dag, self.dag_config, self.release, benchmark.task_id).get_index_task() 
         benchmark >> indexer 
 
     def _get_benchmark(self, benchmark):
         env = {**self.env, **benchmark.get('env', {}), **{"ES_SERVER": var_loader.get_secret('elasticsearch')}, **{"KUBEADMIN_PASSWORD": environ.get("KUBEADMIN_PASSWORD", "")}}
         return BashOperator(
-            task_id=f"{benchmark['name']}",
+            task_id=f"{self.task_group}_{benchmark['name']}",
```

Comment:
can you make this conditional to where we don't prefix the names unless the task_group is not the default `benchmarks`?

Reply:
Yes, makes sense, included a conditional prefix.